conf/application.conf

```diff
@@ -59,7 +58,7 @@ play.http.errorHandler = "uk.gov.hmrc.play.bootstrap.http.JsonErrorHandler"
 # ~~~~~
 # The secret key is used to secure cryptographics functions.
 # If you deploy your application to several instances be sure to use the same key!
-play.crypto.secret = "7qBqeaFQIqZatSX7x8GrqLtmirQUfxvUUID45t1L5zQfjgwKD6YRsvARNzhWD9cn"
+play.http.secret.key = "7qBqeaFQIqZatSX7x8GrqLtmirQUfxvUUID45t1L5zQfjgwKD6YRsvARNzhWD9cn"
```

Comment:
you can remove this - bootstrap provides one for development, and app-config-common provides the real one